packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -380,11 +460,11 @@ contract Attestations is
   )
     external
     view
-    returns (uint64, uint64)
+    returns (uint32, uint32)
   {
     return (
       identifiers[identifier].attestations[account].completed,
-      uint64(identifiers[identifier].attestations[account].issuers.length)
+      identifiers[identifier].attestations[account].requested
```

Comment:
Why do we need to explicitly store the number requested? Wouldn't it be the length of the `issuers` array, plus the number requested in `requests`?

Reply:
`requests` is meant to be ephemeral, so a user could request once, not select, request again, and it would look like they only requested once